README.md

```diff
@@ -24,19 +22,27 @@ See [`navapbc/platform`](https://github.com/navapbc/platform) for other template
 
 ## Installation
 
-To get started using the template application on your project, run the following command to execute the [install script](https://github.com/navapbc/template-application-nextjs/tree/main/template-only-bin/install-template.sh), which clones the template repository, copies the template files to your repository, and then cleans up the template repository:
+To get started using the template application on your project:
 
-```bash
-curl https://raw.githubusercontent.com/navapbc/template-application-nextjs/main/template-only-bin/download-and-install-template.sh | bash -s
-```
+1. Run the download and [install script](https://github.com/navapbc/template-application-nextjs/tree/main/template-only-bin/install-template.sh) in your project's root directory.
 
-Now you're ready to set up the various pieces of your infrastructure.
+    This script will:
 
-## Setup
+    1. Clone the template repository
+    1. Copy the template files into your project directory
+    1. Remove any files specific to the template repository, like this README.
 
-1. [Dependency management](./template-only-docs/set-up-dependency-management.md)
-1. [Deployment](./template-only-docs/set-up-cd.md)
+    ```bash
+    curl https://raw.githubusercontent.com/navapbc/template-application-nextjs/main/template-only-bin/download-and-install-template.sh | bash -s
+    ```
```

Comment:
optional nit, maybe just me but it feels slightly more readable to have the command right after the sentence "run the download and install script" and then have the explainer of what the script will do come afterwards, rather than before.

or an alternative would be to combine the "this script will" clause into the first sentence, like

```
Run the download and install script in your project's root directory, which will:
1. ...
2. ...
3. ...
``bash
curl ...
``

```

Reply:
Good point, updated